frontend/dockerfile/docs/reference.md

```diff
@@ -2031,8 +2031,21 @@ ARG <name>[=<default value>]
 
 The `ARG` instruction defines a variable that users can pass at build-time to
 the builder with the `docker build` command using the `--build-arg <varname>=<value>`
-flag. If a user specifies a build argument that was not
-defined in the Dockerfile, the build outputs a warning.
+flag.
+
+> **Warning**
+>
+> It is not recommended to use build-time variables for passing secrets like
+> GitHub keys, user credentials etc. Build-time variable values are visible to
```

Comment:
```suggestion
> GitHub tokens, user credentials etc. Build-time variable values are visible to
```

Reply:
Reworded slightly more, ptal

Reply:
Ah you just moved the blockquote, looks good then

Reply:
I did explicitly call out buildx actions and public repo thing. Maybe not the best place for it but just to be extra safe I thought?